pages/form.js

```diff
@@ -158,10 +158,10 @@ export default function Home() {
     // Save contact information to google spreadsheet
     saveContactToGoogleSpreadsheet(values);
 
-    if (result.length == 0) {
+    if (response.status != 200) {
```

Comment:
If we don't get a `200/success` it means there was an error, so we'll handle the error apropriately.